pages/birthdays/index.js

```diff
@@ -0,0 +1,226 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Header from "@/components/Header";
+import React, { useState } from "react";
+import { StyledButton } from "@/components/Button/Button.Styled";
+import { StyledButtonBlack } from "@/components/Button/ButtonBlack.Styled";
+import { StyledLink } from "@/components/Link/Link.Styled";
+import { StyledFooter } from "@/components/GeneralStyle/Footer.Styled";
+import { StyledCleanLink } from "@/components/Link/CleanLink.Styled";
+import { StyledLinkBlack } from "@/components/Link/BlackLink.Styled";
+import { StyledAddContactLink } from "@/components/Link/AddContactLink.Styled";
+import { StyledBirthdayLink } from "@/components/Link/BirthdayLink.Styled";
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const StyledBirthdayList = styled.div`
+  max-width: 375px;
+`;
+
+const StyledTitleText = styled.h1`
+  color: white;
+  text-align: center;
+  font-size: 1rem;
+  margin: 1.4rem;
+`;
+
+const StyledIndex = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  text-align: center;
+`;
+
+const StyledBirthdayBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0rem;
+`;
+
+const StyledSingleFriend = styled.div`
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  border-radius: 10px;
+  justify-content: center;
+  width: 20rem;
+  min-width: 0;
+  margin: 0.5rem;
+  padding: 0rem;
+`;
+
+const StyledFriendBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const StyledFriendTextBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0rem 1rem 0rem 1rem;
+`;
```

Comment:
I'm pretty sure you'll find more descriptive and semantic HTML elements for the friends box/maps. 
Grab your time machine and go back a few weeks, when you code the quiz app. 
Maybe an article or something similar would be the right choice? ✨